.github/workflows/publish-demo.yml

```diff
@@ -36,30 +36,20 @@ jobs:
           git-lfs: 'true'
       - name: 🚚 Pull Git LFS files
         run: git lfs pull
-
-      #  Setup Expo CLI action
-      # - name: Setup Expo
-      #   uses: expo/expo-github-action@v5
-      #   with:
-      #     expo-version: 3.x
-      #     expo-username: ${{ secrets.EXPO_CLI_USERNAME }}
-      #     expo-password: ${{ secrets.EXPO_CLI_PASSWORD }}
-      #     expo-cache: true
-
+      - name: Setup Expo
+        uses: expo/expo-github-action@v7
+        with:
+          expo-version: latest
+          eas-version: latest
+          token: ${{ secrets.COMMUNITY_EXPO_ACCESS_TOKEN }}
       - name: 🧶 Install workspace node modules
         if: steps.expo-caches.outputs.yarn-workspace-hit != 'true'
         run: yarn install --frozen-lockfile
       - name: 🔧 Install Expo CLI v3
         run: yarn global add expo-cli@3
-
-      # - name: Publish Expo app
-      #   working-directory: ./apps/native-component-list
-      #   run: expo publish --release-channel=pr-${{ github.event.number }}
-      # # Get Expo link for the comment
-      # - name: Get expo link
-      #   id: expo
-      #   run: echo "::set-output name=path::@community/native-component-list?release-channel=pr-${{ github.event.number }}"
-
+      - name: Publish Expo app
+        working-directory: ./apps/native-component-list
+        run: EXPO_SDK_VERSION=43.0.0 eas update --branch main --message "initial publish"
```

Comment:
Why does this use SDK 43 instead of the latest SDK that matches what NCL uses since it's linked against `main` (I think)?